support-frontend/assets/helpers/existingPaymentMethods/existingPaymentMethods.js

```diff
@@ -17,24 +17,23 @@ export type ExistingPaymentMethodSubscription = {
 
 type ExistingPaymentType = 'Card' | 'DirectDebit';
 
-export type PartialExistingPaymentMethod = {
-  paymentType: ExistingPaymentType;
+export type NotRecentlySignedInExistingPaymentMethod = {
```

Comment:
That's a real mouthful, is there anyway we could come up with something more succinct?

Reply:
i agree, but it is explicit - it is in reaction to https://github.com/guardian/support-frontend/pull/1673#pullrequestreview-219572132 - the shorter name before was more ambiguous